src/monsters.h

```diff
@@ -181,8 +196,8 @@ class Monsters
 
 		std::map<std::string, uint32_t> monsterNames;
 		std::map<MonsterType*, std::string> monsterScriptList;
-		std::map<uint32_t, MonsterType*> monsters;
-		LuaScriptInterface* scriptInterface;
+		std::map<uint32_t, MonsterType> monsters;
```

Comment:
This will be a breaking change - we will be no longer able to create a lua MonsterType metatable by using the uid.